### PR TITLE
Customizations friendly improvements

### PR DESCRIPTION
**Pull request by bartoszpietrzak1994**

### CODE REVIEW

File: spec/Converter/BillingDataConverterSpec.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        $address->getCity()->willReturn('Las Vegas');
+        $address->getPostcode()->willReturn('000001');
+        $address->getStreet()->willReturn('Fremont Street');
+        $address->getProvinceCode()->willReturn('sample_province_code');
+        $address->getProvinceName()->willReturn('sample_province_name');
+        $address->getFirstName()->willReturn('Thomas');
+        $address->getLastName()->willReturn('Shelby');
+        $address->getCompany()->willReturn('Shelby Company Limited');
+
+        $this->convert($address)->shouldReturnAnInstanceOf(BillingDataInterface::class);
```
Comment (GSadee) [inline spec/Converter/BillingDataConverterSpec.php]: You could check here if `BillingData` has proper values

File: spec/Converter/InvoiceChannelConverterSpec.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        $this->shouldImplement(InvoiceChannelConverterInterface::class);
+    }
+
+    function it_converts_channel_to_invoice_channel(ChannelInterface $channel): void
+    {
+        $channel->getCode()->willReturn('en_US');
+        $channel->getName()->willReturn('United States');
+
+        $this->convert($channel)->shouldReturnAnInstanceOf(InvoiceChannelInterface::class);
```
Comment (GSadee) [inline spec/Converter/InvoiceChannelConverterSpec.php]: You could check here if `InvoiceChannel` has proper values

**Comment by GSadee** [comment]
Thanks, Bartosz! :1st_place_medal:

File: src/Resources/config/services/fixtures.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults autoconfigure="true" autowire="true" public="true" />
+
+        <service id="sylius_invoicing_plugin.fixture.example_factory.invoicing_plugin_shop_billing_data" class="Sylius\InvoicingPlugin\Fixture\Factory\ShopBillingDataExampleFactory">
+            <argument id="sylius.repository.channel" type="service"/>
```
Comment (Tomanhez) [inline src/Resources/config/services/fixtures.xml]:
```suggestion
            <argument id="sylius.repository.channel" type="service" />
```
missing space

File: src/Resources/config/services/fixtures.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        </service>
+
+        <service id="sylius_invoicing_plugin.fixture.shop_billing_data_fixture" class="Sylius\InvoicingPlugin\Fixture\ShopBillingDataFixture">
+            <argument id="sylius.manager.channel" type="service"/>
```
Comment (Tomanhez) [inline src/Resources/config/services/fixtures.xml]:
```suggestion
            <argument id="sylius.manager.channel" type="service" />
```

missing space

File: src/Resources/config/services/fixtures.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+            <argument id="sylius_invoicing_plugin.fixture.example_factory.invoicing_plugin_shop_billing_data" type="service"/>
+            <tag name="sylius_fixtures.fixture"/>
```
Comment (Tomanhez) [inline src/Resources/config/services/fixtures.xml]:
```suggestion
            <tag name="sylius_fixtures.fixture" />
```
and here

File: src/Resources/config/services/fixtures.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    <services>
+        <defaults autoconfigure="true" autowire="true" public="true" />
+
+        <service id="sylius_invoicing_plugin.fixture.example_factory.invoicing_plugin_shop_billing_data" class="Sylius\InvoicingPlugin\Fixture\Factory\ShopBillingDataExampleFactory">
+            <argument id="sylius.repository.channel" type="service"/>
+        </service>
+
+        <service id="sylius_invoicing_plugin.fixture.shop_billing_data_fixture" class="Sylius\InvoicingPlugin\Fixture\ShopBillingDataFixture">
+            <argument id="sylius.manager.channel" type="service"/>
+            <argument id="sylius_invoicing_plugin.fixture.example_factory.invoicing_plugin_shop_billing_data" type="service"/>
```
Comment (Tomanhez) [inline src/Resources/config/services/fixtures.xml]:
```suggestion
            <argument id="sylius_invoicing_plugin.fixture.example_factory.invoicing_plugin_shop_billing_data" type="service" />
```

**Pull request merged**